docs/_config.yml

```diff
@@ -51,27 +51,20 @@ settings:
   # Arbitrary multi-level topbar menu
   # IF you don't need this just comment out everything below
   # ----------------------------------
-  topmenu:
-    - title: "Riverscapes"
-      childern:
-        - title: "Model Home"
-          url: index
-        - title: "Riverscapes Home"
-          url: http://riverscapes.xyz
-        - title: "About Riverscapes"
-          url: http://riverscapes.xyz/about.html
-        - title: "Who We Are"
-          url: http://riverscapes.xyz/who-we-are.html
-        - title: "News"
-          url: http://riverscapes.xyz/news.html
-    - title: "RC Tools"
-      url: http://riverscapes.xyz/rc-tools.html
-    - title: "RC Science"
-      url: http://riverscapes.xyz/rc-science.html
-    - title: "RC Data"
-      url: http://riverscapes.xyz/rc-data.html
-    - title: "RC Help"
-      url: http://riverscapes.xyz/rc-help.html
+topmenu:
```

Comment:
The indentation here is not quite right. 

Also we need to make this change in all the `.yml` files, not just the root one

I'll fix it.